finland/router-config.json

```diff
@@ -72,33 +84,78 @@
         },
         "wheelchairAccessibility": {
           "enabled": true
+        },
+        "accessEgress": {
+          "maxStopCountForMode": {
+            "BIKE": 0,
+            "CAR": 0
+          }
         }
       },
       {
-        "modes": "BICYCLE_RENT",
+        "modes": "WALK",
         "walk": {
-          "speed": 1.2,
+          "speed": 1.67,
           "reluctance": 1.8
         },
-        "bicycle": {
-          "speed": 5.55
+        "accessEgress": {
+          "maxStopCountForMode": {
+            "BIKE": 0,
+            "CAR": 0
+          }
         }
       },
       {
-        "modes": "WALK",
+        "modes": "BICYCLE",
         "walk": {
-          "speed": 1.67,
+          "speed": 1.2,
           "reluctance": 1.8
+        },
+        "bicycle": {
+          "speed": 5.55,
+          "rental": {
+            "useAvailabilityInformation": true
+          },
+          "optimization": "SAFEST_STREETS"
+        },
+        "accessEgress": {
+          "maxStopCountForMode": {
+            "BIKE": 0,
+            "CAR": 0
+          }
         }
       },
       {
-        "modes": "BICYCLE_RENT",
+        "modes": "BICYCLE",
         "walk": {
           "speed": 1.67,
           "reluctance": 1.8
         },
         "bicycle": {
-          "speed": 5.55
+          "speed": 5.55,
+          "rental": {
+            "useAvailabilityInformation": true
+          },
```

Comment:
This seems a bit unnecessary since we don't support rental in transfers but I guess the transfer cache gets regenerated with this parameter since we don't have a way to not include a parameter in the transfer cache consideration?